CONTRIBUTING.md

```diff
@@ -85,14 +85,17 @@ Example: `http://localhost:9876/base/test/web-platform-tests/web-animations/anim
     grunt
     # Optional "grunt test" to make sure everything still passes.
     git add -f *.min.js{,.map}
-    git rm .gitignore
+    git rm --ignore-unmatch .gitignore
     git commit -m 'Add build artifacts from '`cat .git/refs/heads/dev`
-    git push HEAD:refs/heads/master
+    git push origin master
```

Comment:
Hmm what happens if their `git remote -v` doesn't show the right value? If it's a fork, I'm guessing it'll point to their own forked repo?